src/tree/settings/CommonProperties.ts

```diff
@@ -52,3 +53,28 @@ export function getCommonGroupIcon(property: CommonProperty | CommonGroupBy): Th
 
     return new ThemeIcon(icon);
 }
+
+export const NonLabelGroupName = l10n.t('others');
```

Comment:
```suggestion
export const NonLabelGroupName = l10n.t('Others');
```
Or maybe "Unlabeled containers"? Though that's not quite right, since they could have other labels.

Reply:
Yeah... either way it's tricky. I'll go with "Others" for now